src/frontend/components/Felleskomponenter/Steg/Steg.tsx

```diff
@@ -165,10 +176,14 @@ const Steg: React.FC<ISteg> = ({ tittel, guide, skjema, gåVidereCallback, child
 
     const { tilbakeKnapp } = tekster().FELLES.navigasjon;
 
+    const dokumentasjonTekster = tekster().DOKUMENTASJON;
     const frittståendeOrdTekster = tekster().FELLES.frittståendeOrd;
 
     const formProgressStegOppsummeringTekst = `${plainTekst(frittståendeOrdTekster.steg)} ${hentNåværendeStegIndex()} ${plainTekst(frittståendeOrdTekster.av)} ${formProgressSteg.length}`;
 
+    const visVedleggOppsummering =
+        vedleggOppsummering && skalVedleggOppsummeringVises(vedleggOppsummering);
```

Comment:
Her ville jeg gjort den sjekken inne i metoden og bare returnert `false` om den var `undefined` eller `null` 

```suggestion
        skalVedleggOppsummeringVises(vedleggOppsummering);
```

Reply:
Dersom jeg flytter håndtering av undefined inn i skalVedleggOppsummeringVises, så vil vedlegg feile  som prop for VedleggOppsummering ettersom den ikke lengre blir sjekket direkte for mulig undefined i filen.

```
{visVedleggOppsummering && (
  <Alert variant="info">
      {plainTekst(dokumentasjonTekster.lastOppSenereISoknad)}
      <VedleggOppsummering vedlegg={vedleggOppsummering} />
  </Alert>
)}
```

![image](https://github.com/user-attachments/assets/78039b11-e063-4938-b8b7-1eb0bb993ad6)

Hva er da best å gjøre i dette tilfellet?